src/firefox/index.ts

```diff
@@ -143,6 +145,26 @@ export default class {
     }
   }
 
+  getKillCmd(pid: number) {
+    return global.platform.win32 ? `taskkill /F /PID ${pid}` : `kill ${pid}`
+  }
+
+  async close() {
+    const processes: Process[] = await find('name', /firefox/i)
+
+    const pointBrowserParentProcesses = processes.filter(
+      p => p.cmd.includes('point-browser') && !p.cmd.includes('tab')
```

Comment:
At least on Linux, I always see two processes running, and killing one does not kill the other. That's why I decided to go with a loop, filtering out any potential browser tabs, as those die when the parent process is killed.